tests/test_commands.py

```diff
@@ -88,7 +88,7 @@ def test_generate_transform_commands_required_input_fields():
         "testsource/testsource-2022-01-02-full-extracted-records-to-index.xml"
     ]
     assert commands.generate_transform_commands(
-        extract_output_files, input_data, "2022-01-02", "test-timdex-bucket"
+        extract_output_files, input_data, "2022-01-02", "test-timdex-bucket", 1
```

Comment:
Related to comments above about where to pull `ETL_VERSION`, perhaps it would allow not modifying all these tests?  Where instead, we could just monkeypatch `ETL_VERSION` (which _itself_ could be a fixture)?

Reply:
See https://github.com/MITLibraries/timdex-pipeline-lambdas/pull/313#discussion_r1869983670. The last two commits undoes the changes to the function signatures for the 'generate' commands and effectively any changes to `lambdas.format_input` handler.